tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -63,6 +63,7 @@ def read_data(filename):
 
 
 def build_dataset(words):
+  global vocabulary_size
```

Comment:
Ok, I will fix it, but the original purpose of me is just to make the original implementation more readable with minimal change.